core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 
 import { OperatorMetadataService } from '../service/operator-metadata/operator-metadata.service';
+import { JointUIService } from '../service/joint-ui/joint-ui.service';
+import { StubOperatorMetadataService } from '../service/operator-metadata/stub-operator-metadata.service';
```

Comment:
Wasn't this imported in the previous PR? because I remember seeing stuboperatormetadatservice being used

Reply:
stuboperatormetadatservice is used another file. This is imported here because in line 15 of this file, we added a line to use it to conveniently test things.

Reply:
JointUIService needs to be important to the parent (in this case, workspace.component.ts file) for workflow-editor.component.ts to use. 

As for the StubOperatorMetadataService, it is only used when we try to test the new-gui without opening the java backend server. Instead of fetching backend metadata, we would use the mock metadata defined in `service/operator-metadata/mock-operator-metadata.data.ts`